lib/chore/worker.rb

```diff
@@ -123,8 +144,19 @@ def handle_failure(item, message, klass, e)
       end
     end
 
-    def perform_job(klass, message)
-      klass.perform(*options[:payload_handler].payload(message))
+    def dupe_detector
+      @dupes ||= DuplicateDetector.new({:servers => Chore.config.dedupe_servers,
+                                        :dupe_on_cache_failure => false})
```

Comment:
Instead of a single connection per consumer instance, we will now have a connection per Worker instance.  In some cases we have job servers with 30+ workers and just a single consumer.  Theoretically we could see more than an order of magnitude increase in the number of connections to the memcached cluster.  While our max is 65k, a connection per worker may actually have an impact on our cluster.  We may want to consider either talking to ops about this or leaving the majority of the dedupe work in the consumer and not the worker.